src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/HelpCommand.kt

```diff
@@ -0,0 +1,21 @@
+package com.noumenadigital.npl.cli.commands.registry
+
+import com.noumenadigital.npl.cli.commands.CommandsRegistry
+import java.io.Writer
+
+data object HelpCommand : CommandExecutor {
+    const val COMMAND_DESCRIPTION = "Display the description for npl-cli commands"
+
+    override val commandName: String = "help"
+
+    override fun execute(output: Writer) {
+        val entries = CommandsRegistry.registeredCommands
+        val padding = entries.keys.maxOf { it.length } + 4
+
+        entries.forEach { registeredCommand ->
+            val name = registeredCommand.key.padEnd(padding)
+            val description = registeredCommand.value
+            output.write("$name$description\n")
+        }
```

Comment:
Here you could now do

```suggestion
        val entries = Command.entries
        val padding = entries.maxOf { it.commandName.length } + 4

        entries.forEach { e ->
            val name = e.commandName.padEnd(padding)
            val description = e.description
        }
```

Which kinda begs the question of what the `CommandRegistry` class is really for, and if we can't inline or move its remaining functionality to more appropriate places.